src/js/components/LoginPage/ButtonStyles.css

```diff
@@ -1,9 +1,10 @@
 .btn-circle {
-  width: 30px;
-  height: 30px;
+  width: 200px;
```

Comment:
was this tested? 200 seems a bit odd?

Reply:
Apologize this has been removed as it was a dummy file not used

Reply:
👍